acct-enricher-job/pom.xml

```diff
@@ -71,6 +71,7 @@
         <scala.logging.version>3.9.0</scala.logging.version>
         <mockito.version>3.2.9.0</mockito.version>
 
+        <ubirch-cassandra-utils.version>1.0-SNAPSHOT</ubirch-cassandra-utils.version>
```

Comment:
Do we need to have the embedded cassandra as part of this pom. Should it not be a dependency that is inherited from the cassandra utils lib?
This also applies for the other pom.

Reply:
I prefer to have it because the embedded cassandra should be compatible with the driver. So if we update the driver in the cassandra utils lib, the embedded cassandra should also be updated as well.
And I saw all services are using the same `EmbeddedBase` by copy & paste. Therefore, it's nice to have a one place to manage it.